### PR TITLE
[🔗 Docs] Fixed Incorrect Migration Link

### PR DESCRIPTION
**Pull request by kadirnar**
I couldn't find it in transformers files. Can you check? Is it true?

### CODE REVIEW

**Comment by kadirnar** [comment]
Which link is correct?
teaser

**Review by amyeroberts — COMMENTED** [review]
Hi @kadirnar, thanks for opening this PR and contributing. 

If you look at the page title in the docs here " Migrate to 🤗 Transformers from `pytorch-transformers` or `pytorch-pretrained-bert`" and the one linked to, they are two unrelated migration pages. 

To update the link, the equivalent page will need to be found.

**Comment by kadirnar** [comment]
Hi @amyeroberts , I am new to transformes library. I wanted to fix this error that I saw in the documentation. Because the link is broken.


The documentation page MIGRATION doesn't exist in v4.30.0, but exists on the main version. Click here to redirect to the main version of the documentation.

I looked in transformers documentation(https://huggingface.co/docs) and couldn't find it. Can you help?

File: README.md [hunk 1]
```diff
@@ -518,7 +518,7 @@ These implementations have been tested on several datasets (see the example scri
 | [Training and fine-tuning](https://huggingface.co/docs/transformers/training) | Using the models provided by 🤗 Transformers in a PyTorch/TensorFlow training loop and the `Trainer` API |
 | [Quick tour: Fine-tuning/usage scripts](https://github.com/huggingface/transformers/tree/main/examples) | Example scripts for fine-tuning models on a wide range of tasks |
 | [Model sharing and uploading](https://huggingface.co/docs/transformers/model_sharing) | Upload and share your fine-tuned models with the community |
-| [Migration](https://huggingface.co/docs/transformers/migration) | Migrate to 🤗 Transformers from `pytorch-transformers` or `pytorch-pretrained-bert` |
+| [Migration](https://huggingface.co/docs/accelerate/basic_tutorials/migration) | Migrate to 🤗 Transformers from `pytorch-transformers` or `pytorch-pretrained-bert` |
```
Comment (amyeroberts) [inline README.md:521]: Interesting! Indeed, it seems that the page was deleted in #23049. I'd suggest just removing the link altogether 

```suggestion
```

**Comment by kadirnar** [comment]
Thank you for the help💖

**Review by sgugger — APPROVED** [review]
Thanks!

**Pull request merged**